manifest.yml

```diff
@@ -28,7 +28,6 @@ terraform_binaries:
 - name: tofu
   version: 1.6.1
   source: https://github.com/opentofu/opentofu/archive/refs/tags/v1.6.1.zip
-  url_template: https://github.com/opentofu/opentofu/releases/download/v${version}/tofu_${version}_${os}_${arch}.zip
```

Comment:
Context for future us.

Removing `url_template` for tofu would have caused issues in cloud-service-broker for any version below v1.0.0.
This is so because [previously when `url_template` was missing it defaulted to the constant HashicorpUrlTemplate](https://github.com/cloudfoundry/cloud-service-broker/blob/b080dd439e50581fcd5389530b1488d766efb495/internal/brokerpak/brokerpakurl/brokerpakurl.go#L13C31-L13C117).

However, https://github.com/cloudfoundry/cloud-service-broker/pull/975 put some logic in place to treat `url_template` differently depending if it's a provider or the tofu binary itself.

Since such logic was added in v1.0.0, the additional bumps included in this PR are merely complementary and everything should have worked without them too.